account_move_fiscal_month/models/account_move.py

```diff
@@ -12,8 +12,15 @@ class AccountMove(models.Model):
 
     date_range_fm_id = fields.Many2one(
         comodel_name='date.range', string="Fiscal month",
+        domain=lambda self: self._get_date_range_fm_domain(),
```

Comment:
why the lambda?

Reply:
The lambda is used to be able to inherit from ```_get_date_range_fm_domain```.